choose/command.go

```diff
@@ -106,6 +106,8 @@ func (o Options) Run() error {
 		itemStyle:         o.ItemStyle.ToLipgloss(),
 		selectedItemStyle: o.SelectedItemStyle.ToLipgloss(),
 		numSelected:       currentSelected,
+		hasTimeout:        o.HasTimeout(),
+		timeout:           o.Timeout,
```

Comment:
I believe for `gum choose` we should either make it select:
1. The first option always
2. The option the user's cursor is on when the timeout hits
3. Exit with status 1 to indicate timeout

Currently, the `gum confirm 1 2 3 --timeout 2s` will timeout with status 0 (success) but not have any output which could be misleading, so we should either exit with status 1 or return some output.

Reply:
I think you made a typo in the command. It is supposed to be choose, right ? If so, I would agree on selecting the first/selected option when timeout hits

Reply:
Yes typo! I meant gum choose!